<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MinhoUDPCOM</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rs.qrc">
    <normaloff>:/Pictures/simboloMinhoTeam.png</normaloff>:/Pictures/simboloMinhoTeam.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="8" column="0" colspan="3">
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="4">
     <widget class="QComboBox" name="cb_serial">
      <property name="font">
       <font>
        <family>Cabin</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="btRefresh">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Cabin</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border:1px solid black;
background-color: rgb(0, 0, 168);
color:white;
border-radius:10%;
}

QPushButton:hover{
border:4px solid black;
}</string>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="btCloseSerial">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Cabin</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border:1px solid black;
background-color: rgb(170, 0, 0);
color:white;
border-radius:10%;
}

QPushButton:hover{
border:4px solid black;
}</string>
      </property>
      <property name="text">
       <string>Close Serial Port</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="tbMyPort">
      <property name="font">
       <font>
        <family>Cabin</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>1234</string>
      </property>
      <property name="placeholderText">
       <string>My Port</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLineEdit" name="tbMyIp">
      <property name="font">
       <font>
        <family>Cabin</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>My Ip Adress</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="2">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLineEdit" name="tbDestIp">
      <property name="font">
       <font>
        <family>Cabin</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Destination Ip Adress</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Cabin</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Destination Ip Adress:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="tbDestPort">
      <property name="font">
       <font>
        <family>Cabin</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>1235</string>
      </property>
      <property name="placeholderText">
       <string>Destination Port</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="btOpenSerial">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Cabin</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border:1px solid black;
background-color: rgb(14, 145, 18);
color:white;
border-radius:10%;
}

QPushButton:hover{
border:4px solid black;
}</string>
      </property>
      <property name="text">
       <string>Open Serial Port</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Cabin</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Destination Port:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Cabin</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>My Ip Adress:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Cabin</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>My Port:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QPushButton" name="btOpenUDP">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Cabin</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border:1px solid black;
border-radius:10%;
background-color: rgb(14, 145, 18);
color:white;
}

QPushButton:hover{
border:4px solid black;
}</string>
      </property>
      <property name="text">
       <string>Open UDP Comunication</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="2">
     <widget class="QLineEdit" name="tbSend">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <widget class="QPushButton" name="btSendUDP">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Cabin</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border:1px solid black;
background-color: rgb(14, 145, 18);
color:white;
border-radius:10%;
}

QPushButton:hover{
border:4px solid black;
}</string>
      </property>
      <property name="text">
       <string>Send UDP</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="rs.qrc"/>
 </resources>
 <connections/>
</ui>
